gate-api-tck/src/main/kotlin/com/netflix/spinnaker/gate/api/test/GateFixture.kt

```diff
@@ -39,3 +49,24 @@ inline fun <PF, reified F> TestContextBuilder<PF, F>.gateFixture(
     }
   }
 }
+
+@TestConfiguration
```

Comment:
Two copies of this, plus [RedisTestConfig](https://github.com/spinnaker/gate/blob/b681e07482038adb0c8022094005cf13f69ee02c/gate-web/src/test/groovy/com/netflix/spinnaker/gate/config/RedisTestConfig.groovy) seems like a lot. Is there any reasonable way to share them?

Reply:
It is part of code in different modules and packages, and using EmbeddedRedis as test source for redis. So, IMO it would be difficult and may not be helpful beyond these testcases.